deploy/README.md

```diff
@@ -14,3 +14,56 @@ The just defines the infrastructure for the express app. This includes:
   amount of pods to manage the load
 - [Config](https://kubernetes.io/docs/concepts/configuration/configmap/) - configurable values for the app
 - [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) - secret config values for the app
+
+# Alerts
+The following sections are short runbooks to explain what each alert means, and actions that could be taken toremedy the issue.
+
+
+# SlowResponse
+This alert fires when, over a 3 minute period, more than 1% of requests were slower than 5 seconds. In other words the p99 latency is > 5s.
+This means that some users are experiencing slow loading of the website. 
+Action could include:
+- Reviewing the ingress dashboard - the link is in the Slack message.
+- Checking the pods e.g.
+```
+NSP=care-arrangement-plan-prod
+kubectl -n $NSP get posd -owide
+```
+
+# SlownessOutage
+This alert fires when, over a 3 minute period, more than 10% of requests were slower than 7.5 seconds.
+In other words the p90 latency is > 7.5s.
+This means that a significant percentage of users are experiencing slow loading of the website.
+Check the actions under the slow response section.
+Additional action could include:
+- Restart the pods with `kubectl rollout restart deployment/$NSP -n $NSP`
```

Comment:
This should be 

`kubectl rollout restart deployment/$NSP-deployment -n $NSP`

then it will match the `name` value in the deployments. e.g. https://github.com/ministryofjustice/care-arrangement-plan/blob/1ad30aaa3c845b32d9cce40ee9a1394e69353d38/deploy/development/deployment.tpl.yml#L4C9-L4C37